### DISCUSSION TITLE
Heco/HTX issued mapping ring on Heco and can’t bridge?

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#61] [Ideas]
There is a Ring token on Heco which is created/backed/issued by Heco bridge/HTX, but the token bridge seems broken and users do not know how to bridge back to Ethereum through HTX withdrawal/deposit.

HTX helper center might need to be contacted? This is just to record the issues reported from community channels.

**Comment by SasoLithops** [comment]
We should contact also Evolution Land players on Dawning (Heco) once solution is offered if they got some Ring stuck there.
